gtsfm/multi_view_optimizer.py

```diff
@@ -80,8 +83,15 @@ def create_computation_graph(
 
         ba_result_graph = self.ba_optimizer.create_computation_graph(ba_input_graph)
 
+        if gt_poses_graph is not None:
+            self.metrics_graph = dask.delayed(metrics.save_averaging_metrics)(
+                i2Ui1_graph, wRi_graph, wti_graph, gt_poses_graph, 'metrics')
```

Comment:
we recently merged another branch that saves to "result_metrics", can we use the same one here?

https://github.com/borglab/gtsfm/pull/94/files#diff-e715d8778a0c656f504e46468a51441480965c02932c4b2256e30a2239335e4eR388